demo/simple-walkthrough/Getting Started with Dataflux Dataset for PyTorch with Google Cloud Storage.ipynb

```diff
@@ -647,6 +642,130 @@
+        "    project_name=PROJECT_ID,\n",
+        "    bucket_name=\"dataflux-demo-public\",\n",
+        "    config=dataflux_mapstyle_dataset.Config(\n",
+        "        prefix=\"diffusiondb-large/\",\n",
+        "        num_processes=1,\n",
+        "    ),\n",
+        ")\n",
+        "t1 = time()\n",
+        "print(f\"With sequential listing, it takes {t1-t0} seconds to list the dataset \"\n",
+        "      f\"of {len(dataset.objects)} objects.\")"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "3kwqnc6BTarG",
+        "outputId": "d32983e5-a497-4629-f50d-060348b5536f"
+      },
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "With sequential listing, it takes 146.16410398483276 seconds to list the dataset of 1999002 objects.\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "t0 = time()\n",
+        "dataset = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(\n",
+        "    project_name=PROJECT_ID,\n",
+        "    bucket_name=\"dataflux-demo-public\",\n",
+        "    config=dataflux_mapstyle_dataset.Config(\n",
+        "        prefix=\"diffusiondb-large/\",\n",
+        "        num_processes=5,\n",
+        "    ),\n",
+        ")\n",
+        "t1 = time()\n",
+        "print(f\"With fast listing, it takes {t1-t0} seconds to list the dataset of \"\n",
+        "      f\"{len(dataset.objects)} objects.\")"
+      ],
+      "metadata": {
+        "id": "QD3wRLO0MWAr",
+        "outputId": "d4e364d4-2f94-4e5a-8148-83647d534d20",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 10,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "With fast listing, it takes 69.39731621742249 seconds to list the dataset of 1999002 objects.\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "\n",
+        "os.cpu_count()"
+      ],
+      "metadata": {
+        "id": "b7p5n8p_ZJ7i",
+        "outputId": "daf47666-cf41-4310-a3d3-84abd573acba",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 12,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "2"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 12
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "You can see that with 2 available CPUs from the Colab runtime, fast listing can speed up the listing time by 2x from 146 seconds to 69 seconds."
```

Comment:
"Using the two available CPUs in the Colab runtime, fast listing can improve performance by 2x (from 146 to 69 seconds)."

Reply:
done.